src/ziggurat/messaging/consumer.clj

```diff
@@ -33,8 +38,8 @@
   (lb/ack ch delivery-tag))
 
 (defn process-message-from-queue [ch meta payload topic-entity processing-fn]
-  (let [delivery-tag (:delivery-tag meta)
-        message-payload      (convert-and-ack-message ch meta payload false topic-entity)]
+  (let [delivery-tag    (:delivery-tag meta)
```

Comment:
[Not part of the original problem] But, we can publish-to-deadset even for `process-message-from-queue` instead of re-queuing the message.

Reply:
done